docs/how_to_use.rst

```diff
@@ -80,6 +80,31 @@ There are 2 types of tests included in pytest-splunk-addon.
 
             -m  splunk_searchtime_cim
 
+----------------------
+
+There are some optional arguments for pytest-splunk-addon
```

Comment:
Can we change it to
```
The following optional arguments are available to modify the default settings in the test cases.
```
@bhargav-nariyani-crest  Your thougths?

Reply:
Sounds good.

Reply:
Done.